src/UrdFactory.sol

```diff
@@ -10,6 +10,12 @@ import {UniversalRewardsDistributor} from "./UniversalRewardsDistributor.sol";
 /// @custom:contact security@morpho.org
 /// @notice This contract allows to create UniversalRewardsDistributor (URD) contracts, and to index them easily.
 contract UrdFactory {
+    /* STORAGE */
+
+    mapping(address => bool) public isUrd;
```

Comment:
so we are not storing the creator (msg.sender) address

Reply:
No we came to the conclusion that it was not necessary as we can easily retrieve the address of the deployer and thus it adds unnecessary logic

Reply:
(which results in unnecessary complexity on Etherscan when using this getter)

Reply:
(or unnecessary complexity when using this getter on Etherscan)

Reply:
Clear!